systemd/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+include(GNUInstallDirs)
+
+configure_file(${CMAKE_SOURCE_DIR}/systemd/ceph-osd@.service.in
+  ${CMAKE_BINARY_DIR}/systemd/ceph-osd@.service @ONLY)
```

Comment:
sorry, i missed that you do `install()` the created file. then please just use `${CMAKE_CURRENT_BINARY_DIR}` here. like
```cmake
configure_file(${CMAKE_SOURCE_DIR}/systemd/ceph-osd@.service.in
  ${CMAKE_CURRENT_BINARY_DIR}/ceph-osd@.service @ONLY)
```